projects/rocprofiler-systems/source/lib/core/gpu.cpp

```diff
@@ -138,6 +139,10 @@ query_rocm_agents()
             cur_agent.model_name           = std::string(_agent->model_name);
             cur_agent.vendor_name          = std::string(_agent->vendor_name);
             cur_agent.product_name         = std::string(_agent->product_name);
+
+            auto json_str        = agent_info::to_json_string(*_agent);
+            cur_agent.agent_info = json_str;
```

Comment:
```suggestion
            cur_agent.agent_info = agent_info::to_json_string(*_agent);
```
nit: Was the intermediate variable required?

Reply:
Not really.